charts/enterprise/traefik/templates/middlewares/rewrite-headers.yaml

```diff
@@ -0,0 +1,15 @@
+{{- range $index, $middlewareData := .Values.middlewares.rewriteResponseHeaders }}
+---
+apiVersion: traefik.io/v1alpha1
+kind: Middleware
+metadata:
+  name: {{ ternary (printf "%v-%v" $.Release.Name $middlewareData.name) $middlewareData.name $.Values.ingressClass.enabled }}
+  namespace: {{ $.Release.Namespace }}
+spec:
+  plugin:
+    rewriteResponseHeaders:
+      rewrites:
+        - header: {{ $middlewareData.header }}
+          regex: {{ $middlewareData.regex }}
+          replacement: {{ $middlewareData.replacement }}
+{{- end -}}
```

Comment:
Why is that limited to a single header rewrite per middleware?
See examples of customResponseHeaders

Reply:
Fixed. Should be good now but please double check 😅